dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -19,15 +19,16 @@ ENV NJOBS="2" \
     OPAMROOTISOK="true"
 
 # Base opam is the set of base packages required by Coq
+# `num` does not have a dependency as it varies on the compiler version.
 ENV COMPILER="4.02.3" \
-    BASE_OPAM="num ocamlfind jbuilder ounit"
+    BASE_OPAM="num ocamlfind.1.8.0 jbuilder.1.0+beta20 ounit.2.0.8"
```

Comment:
While you're at it move BASE_OPAM to the next ENV directive, immediately before it's used.

Reply:
Moved and tweaked a bit.